### DISCUSSION TITLE
Android auto support

### DISCUSSION THREAD

**Discussion by G33ck01** [organicmaps/organicmaps#1058] [Ideas]
I've been searching for a tracking free alternative to Google maps for ages and this seems to be a really promising one. A feature that I'd miss is the android auto support, in order to use it in my car for navigation.

Keep up the good work!

**Comment by weilmd** [comment]
Yes, indeed.  Working with Android Auto would be a GREAT improvement.  Currently, you get Sygic or Google Maps.  My issue is looking for a offline GPS, and both of the mentioned apps use small downloadable offline areas, or charge for mapping and are not updated frequently.

**Comment by MrX-ua** [comment]
Would be indeed awesome to get Android Auto support. It's a great`offline map app, the only one I use and would love to be able to use in the car as well

**Reply by ogarm** [reply]
Hi, I see Organic Maps in my iPhone but I can't find it in my Samsung phone (for work). It would be awesome to have integration with Android Auto. It is still just an idea? Is in development? It's been deprecated?

Thanks

**Reply by biodranik** [reply]
1. Does your Samsung phone has Google Play installed? Organic Maps is available there.
2. Please follow #3806

**Reply by ogarm** [reply]
Hi, yes it's a S10 with Android 12 with October security update. I just want AA support not AAOS as in the mentioned thread, but I can see it should work also.
I've downloaded the latest stable release but it doesn't show up in the AA list of available apps.

**Reply by ogarm** [reply]
Well after reading the whole thread I don't know if it's yet in production or in development 😅

**Reply by AndrewShkrob** [reply]
It's in development. In the very beginning 😄

**Reply by Malachiel87** [reply]
Cant wait!

**Reply by cateyes99** [reply]
This is a must-have. Without this, the app loses over the half of its value. I'm so keen to see it in action. Wonder how is this feature going now?

**Reply by AndrewShkrob** [reply]
You can check the status here:
https://github.com/organicmaps/organicmaps/issues/4096
https://github.com/organicmaps/organicmaps/issues/5404

**Comment by PinguDEV-original** [comment]
Close this pls, there is Android Auto support!

**Comment by biodranik** [comment]
Android Auto was implemented in the beginning of this year: https://organicmaps.app/news/2024-01-10/android-auto-is-finally-available-in-google-play/

**Reply by yekibud** [reply]
Does this mean it's also available for Android automotive?

**Reply by biodranik** [reply]
Android Automotive OS is not supported yet.

**Reply by yekibud** [reply]
Okay, has there been any investigation into this and is it on the roadmap somewhere?  Since you've already added some code for Android auto, isn't it as simple as publishing here https://developer.android.com/training/cars/distribute.  If there's Dev work, any guidance on where to get started? I was just going to look at the automotive emulator and the Google development guides see if I could get anywhere with either organicmaps or osmand.  But I'm sure I can't do the publishing part if that's all there is.

**Reply by biodranik** [reply]
It would be great if you (or someone else) could investigate and then describe what should be done (and what are caveats) to support Automotive OS.

**Comment by weilmd** [comment]
Yes it is... At least on my car it goes directly to the car audio/video
panel..

**Reply by AndrewShkrob** [reply]
No, it is not. Android Automotive is an OS for cars while Android Auto is just a "screen mirroring" from mobile phone. OM doesn't support Android Automotive yet.
